src/workspace-data/data-table/shared/DataTable.test.ts

```diff
@@ -50,45 +58,12 @@ jest.mock('react-virtualized', (): ReactVirtualizedExports => {
   };
 });
 
-// Returns a filtered subset of |entities| if a filter is included - more than one page
-// Else 100-item pages depending on page number
-const getPage = jest
-  .fn()
-  // eslint-disable-next-line @typescript-eslint/no-unused-vars
-  .mockImplementation((signal, entityType, queryOptions: { pageNumber; columnFilter }, entityMetadata) => {
-    if (queryOptions.columnFilter) {
-      return Promise.resolve({
-        results: _.filter((s: { attributes: { attr: string } }) => s.attributes.attr === 'even', entities),
-        resultMetadata: { filteredCount: 125, unfilteredCount: 250, filteredPageCount: 2 },
-      });
-    }
-    if (!queryOptions.pageNumber || queryOptions.pageNumber === 1) {
-      return Promise.resolve({
-        results: entities.slice(0, 100),
-        resultMetadata: { filteredCount: 250, unfilteredCount: 250, filteredPageCount: 3 },
-      });
-    }
-    if (queryOptions.pageNumber === 2) {
-      return Promise.resolve({
-        results: entities.slice(100, 200),
-        resultMetadata: { filteredCount: 250, unfilteredCount: 250, filteredPageCount: 3 },
-      });
-    }
-    return Promise.resolve({
-      results: entities.slice(200),
-      resultMetadata: { filteredCount: 250, unfilteredCount: 250, filteredPageCount: 3 },
-    });
-  });
-
-const mockDataProvider: DeepPartial<DataTableProvider> = {
-  getPage,
-  features: {
-    supportsFiltering: true,
-    supportsRowSelection: true,
-  },
-};
+const mockDataProvider = new EntityServiceDataTableProvider('test-namespace', 'test-workspace');
```

Comment:
This doesn't need to be mocked since we're mocking the underlying Ajax call: `paginatedEntitiesOfType`.